x/oracle/client/cli/txUpdateOracleInfo.go

```diff
@@ -0,0 +1,60 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdUpdateOracleInfo() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-oracle-info",
+		Short: "update an oracle information",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			oracleAddress := clientCtx.GetFromAddress().String()
+
+			endpoint, err := cmd.Flags().GetString(flagOracleEndpoint)
+			if err != nil {
+				return fmt.Errorf("failed to get oralce end point")
+			}
+
+			oracleCommissionRateStr, err := cmd.Flags().GetString(flagOracleCommRate)
+			if err != nil {
+				return fmt.Errorf("failed to get oralce commission rate")
+			}
+
+			if len(oracleCommissionRateStr) == 0 {
+				return fmt.Errorf("oracleCommissionRate is empty")
+			}
+
+			oracleCommissionRate, err := sdk.NewDecFromStr(oracleCommissionRateStr)
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgUpdateOracleInfo(oracleAddress, endpoint, oracleCommissionRate)
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	cmd.Flags().String(flagOracleEndpoint, "", "oracle's endpoint")
+	cmd.Flags().String(flagOracleCommRate, "", "oracle's commission rate")
```

Comment:
please mark as required field as done in register-oracle

Reply:
It has not been added as part of this section. This is because it was thought that this CLI should work even if only one of the two values is entered. 
I thought about a part that handles when there are no values, but I excluded it for now because I thought this process would become more complicated as the number of update info increases later. wdyt?

Reply:
I got it. Then, I think `UpgradeOracleInfo` should update commission rate only when the `msg.CommissionRate` is not nil.

Reply:
When I think about it again,  it is better to apply `MarkFlagRequired` because the commission rate returns an error when it is empty. 
And I made some modifications to the keeper function. If there is no change in the input information, an error is returned in order not to update `UpdateTime`. And, commission rate only update when the commission rate is changed.

Reply:
Thank you for your suggestions. This way, the user seems to be able to request the oracle commission as a nil value as well. Please check [d35db47](https://github.com/medibloc/panacea-core/pull/540/commits/d35db4770e1fb267b350286643ee78ea6351cfe9).